axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/ProjectTaskBusinessProjectServiceImpl.java

```diff
@@ -562,4 +567,47 @@ protected BigDecimal convertTimesheetLineDurationToProjectTaskUnit(
 
     return convertedDuration;
   }
+
+  /**
+   * update project task values for reporting
+   *
+   * @param projectTask
+   * @throws AxelorException
+   */
+  protected void computeProjectTaskReporting(ProjectTask projectTask) throws AxelorException {
+    if (projectTask.getUpdatedTime().signum() <= 0 || projectTask.getSoldTime().signum() <= 0) {
+      throw new AxelorException(
+          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
+          String.format(
+              I18n.get(BusinessProjectExceptionMessage.PROJECT_TASK_UPDATE_REPORTING_VALUES_ERROR),
+              projectTask.getFullName()));
+    }
+
+    BigDecimal percentageOfProgression =
+        projectTask
+            .getSpentTime()
+            .multiply(new BigDecimal("100"))
+            .setScale(BIG_DECIMAL_SCALE, RoundingMode.HALF_UP);
```

Comment:
I think you can remove the setScale() here and add it only at the divide, this way we will avoid any rounding errors.